diff --git a/node_modules/react-native-iap/src/types/index.ts b/node_modules/react-native-iap/src/types/index.ts
index 46f0603..773e4d9 100644
--- a/node_modules/react-native-iap/src/types/index.ts
+++ b/node_modules/react-native-iap/src/types/index.ts
@@ -271,9 +271,9 @@ export type RequestSubscription =
 
 declare module 'react-native' {
   interface NativeModulesStatic {
-    RNIapIos: IosModuleProps;
-    RNIapIosSk2: IosModulePropsSk2;
-    RNIapModule: AndroidModuleProps;
-    RNIapAmazonModule: AmazonModuleProps;
+    // RNIapIos: IosModuleProps;
+    // RNIapIosSk2: IosModulePropsSk2;
+    // RNIapModule: AndroidModuleProps;
+    // RNIapAmazonModule: AmazonModuleProps;
   }
 }
